src/clj_chrome_devtools/impl/connection.clj

```diff
@@ -91,24 +91,29 @@
       (throw (ex-info "No debuggable pages found"
                       {:pages pages}))))
 
+(defn- make-ws-client [ & [{:keys [idle-timeout max-msg-size-mb]
```

Comment:
Should `make-ws-client` be public? If the user can construct their own and give it to `connect` as parameter.

Reply:
Good idea. Just pushed a change to make it public along with doc string, project version and change log updates.